assets/models/system/microsoft-orca-2-7b/spec.yaml

```diff
@@ -0,0 +1,26 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: microsoft-Orca-2-7b
+path: ./
+properties:
+  SHA: 60e31e6bdcf582ad103b807cb74b73ee1d2c4b17
+  datasets: Publicly available sources
+  inference-recommended-sku: Standard_NC12s_v3, Standard_NC24s_v3, Standard_NC24rs_v3, Standard_NC8as_T4_v3, Standard_NC16as_T4_v3, Standard_NC64as_T4_v3, Standard_ND40rs_v2, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
+  languages: en
+tags:
+  Preview: ''
+  inference_compute_allow_list:
+  - Standard_NC12s_v3
+  - Standard_NC24s_v3
+  - Standard_NC24rs_v3
+  - Standard_NC8as_T4_v3
```

Comment:
Let's add only the ones we have verified on.

Reply:
Done